client/components/users/UserProfile.jsx

```diff
@@ -58,17 +58,21 @@ function UserProfile({ loggedInUser = { id: 1, username: "admin" } }) {
           <UserStats loggedInUser={loggedInUser} />
           <UserRankings loggedInUser={loggedInUser} />
         </Grid>
-        <Grid
+        {/* <Grid
           item
           xs={5}
           container
           direction="column"
           display="flex"
           spacing={2}
         >
-          <UserQuizHistoryTaken />
-          <UserQuizHistoryCreated />
-        </Grid>
+
+        </Grid> */}
```

Comment:
I think you may have just forgotten to take this out, but take a look and remove if not needed :-)